### ISSUE TITLE
Regarding licence of model file and dataset. 

### ISSUE THREAD

**Issue by selva18** [YadiraF/PRNet#28]
hi YadiraF,
I have gone through the repository and found that the license type used in this repository is "MIT".
1.Is the model file also covered under MIT license?
2.With your knowledge, any idea about the licensing of datasets you used?

**Comment by kylemcdonald** [comment]
The training data is from [3DDFA](http://www.cbsr.ia.ac.cn/users/xiangyuzhu/projects/3DDFA/main.htm), which does not itself include a license, but uses other models and datasets:

- [BFM](https://faces.dmi.unibas.ch/bfm/bfm2017.html) which includes a non-commercial use stipulation: "NON-COMMERCIAL USE: The license granted is for internal, non-commercial research, evaluation or testing purposes only. Any use of the DATA or its contents to manufacture or sell products or technologies (or portions thereof) either directly or indirectly for any direct or indirect for-profit purposes is strictly prohibited."
- [LPFW](https://neerajkumar.org/base/databases/lfpw/) which includes a non-commercial use stipulation: "The database is made available only for non-commercial use."

Furthermore, it includes datasets like [Helen](http://www.ifp.illinois.edu/~vuongle2/helen/) which use training data from Flickr users that is not necessarily licensed for use as training data.

In short, it's not possible that the model could be licensed for free use. At best, it's free for research purposes only.

**Comment by kidapu** [comment]
@kylemcdonald @YadiraF 

Hi, I consider to use PRNet for commercial use by re-traning model with available dataset.

> BFM 

BFM is included in training data set as "300W-LP" .
In [this url](http://www.cbsr.ia.ac.cn/users/xiangyuzhu/projects/3DDFA/main.htm), 
there is also "300W-3D-Face", which seems to no need to use BFM.
If I re-trainig PRNet with this data, Can I use PRNet for commercial use?

> LPFW ( LFPW? )

I can't find LPFW( LFPW? ) in [thesis document](http://openaccess.thecvf.com/content_ECCV_2018/papers/Yao_Feng_Joint_3D_Face_ECCV_2018_paper.pdf).
Could you tell me where does PRNet use LPFW ?
